patches/chrome-browser-ui-views-location_bar-location_bar_view.cc.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
+index bc8c76a21935f392d40aa0ac1798d8e9b3cdbc7b..99aabe9bc85a80394728cb694e9e6ac80cf6ccf4 100644
+--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
++++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
+@@ -253,10 +253,12 @@ void LocationBarView::Init() {
+     page_action_icons_.push_back(intent_picker_view_ =
+                                      new IntentPickerView(browser_, this));
+ #endif
++#if !defined(BRAVE_CHROMIUM_BUILD)
+   if (browser_) {
+     page_action_icons_.push_back(
+         star_view_ = new StarView(command_updater(), browser_, this));
```

Comment:
I'm not sure if it's possible but I wonder if we did chromium_src override and replaced StarView with something else like NullView. Check if you can modify StarView to do nothing, and if so make a new NullView that you replace it with without patching.